### ISSUE TITLE
graphs are not working after upgrade to 3.0

### ISSUE THREAD

**Issue by HonkXL** [piwik/piwik#11075]
Since upgrading to Piwik 3.0.0 the graphs are no longer updating the displayed data. The old datas before the upgrade are still displayed, but no actual visits.
But the visits are tracked by Piwik!
Look to the screenshot: the graph says that I had no visitors the last 4 days, but below the text shows that I had visitors in the last 24 hours...

Maybe it matters: I use PHP 7.1, but it worked with Piwik 2 & PHP 7.1.

<img width="157" alt="piwik-graph" src="https://cloud.githubusercontent.com/assets/16715040/21469123/2787f004-ca3b-11e6-9cdf-320388832629.png">

**Comment by trollverk** [comment]
Same here on PHP 5.6.28.

**Comment by HonkXL** [comment]
OK, so it seems not to be the PHP version and not the ecommerce-plugin, because I don't use this.

**Comment by mattab** [comment]
Question:

* What is the value of "Archive reports when viewed from the browser" in Administration > General Settings?

* If you run Piwik [cron archiving](http://piwik.org/docs/setup-auto-archiving/) does it output any error?

**Comment by HonkXL** [comment]
Thank you - I think I found the reason for this problem: My webserver is using Plesk as configuration interface. PHP for the domain with Piwik ist configured to 7.1, but default PHP of the server is 5.4. So "console core:archive" brings an error message now. I changed my cron to
/opt/plesk/php/7.1/bin/php /path-to-piwik/console core:archive
and everything is ok now.
I think this could be added to the FAQs?
